### PR TITLE
Cache locations in memory during contracts load

### PR DESCRIPTION
**Pull request by catherinedevlin**
On the contract file I was benchmarking on (SBA 17 Contracts.csv), this sped the overall load up by 38%.

### CODE REVIEW

File: usaspending_api/settings.py [hunk 1]
```diff
@@ -845,3 +845,23 @@
     "original_loan_subsidy_cost": "original_loan_subsidy_cost",
     "business_funds_indicator": "business_funds_indicator"
 }
+
+# If caches added or renamed, edit clear_caches in usaspending_api/etl/helpers.py
+CACHES = {
```
Comment (bsweger) [inline usaspending_api/settings.py]: @catherinedevlin quick question...are we using the recipients and summary awards caches (want to make sure I understand how this is working...I see the explicit code for using the location cache).

if we haven't implemented the recipients/awards caching yet, is it worth putting a note somewhere to our future selves that this is a to do when we return to this story?

File: usaspending_api/etl/helpers.py [hunk 2]
```diff
@@ -1,9 +1,18 @@
...
 from django.db.models import Q
+from django.core.cache import caches, CacheKeyWarning
 
 from usaspending_api.references.models import Agency, Location, RefCountryCode
 
+warnings.simplefilter("ignore", CacheKeyWarning)
+
+
+def clear_caches():
+    for cache in ['locations', 'recipients', 'summary-awards']:
```
Comment (bsweger) [inline usaspending_api/etl/helpers.py]: A very minor nit (and I know this is coming from the loader code itself)...the name `summary-award` is still hanging around from the days before we renamed the `SummaryAward` model to just `Award`. Do you think it's worth starting to make our code match that new name where we can?

File: usaspending_api/settings.py (hunk 1 above)
Comment (destiejo) [inline usaspending_api/settings.py]: I'd echo this - I know in contracts and assistance we build a few lists (IE txn) - would we need to insert a "recipient_cache = caches['recipients']" for the two loaders? 

Wasn't sure if the default cache just automatically stored everything you've already pulled! I know we clear the caches before you run the rest of the code in the loader too.

Comment (catherinedevlin) [inline usaspending_api/settings.py]: I suppose it's clearer just to delete mention of the unused caches.  I stopped in the middle of implementing them...

File: usaspending_api/etl/helpers.py (hunk 2 above)
Comment (catherinedevlin) [inline usaspending_api/etl/helpers.py]: good point!  Taking out that cache anyway for now, but if we ever re-do we can use a better name

**Pull request merged**